### PR TITLE
Fourmolu fixup

### PR DESCRIPTION
**Pull request by kostmo**
Switch to `fourmolu-0.13` and reformat all source code.

### CODE REVIEW

File: src/Swarm/Language/Pipeline.hs [hunk 1]
```diff
@@ -40,12 +40,12 @@ import Witch
 --   pipeline.  Put a 'Term' in, and get one of these out.
 data ProcessedTerm
   = ProcessedTerm
+      -- | The elaborated + type-annotated term, plus types of any embedded definitions
       TModule
-      -- ^ The elaborated + type-annotated term, plus types of any embedded definitions
+      -- | Requirements of the term
```
Comment (byorgey) [inline src/Swarm/Language/Pipeline.hs:45]: I think Haddock chokes on this.  But once we rebase this PR on top of #1308 it should be fine.

**Comment by byorgey** [comment]
I tried updating to `fourmolu-0.13.0.0` but sadly it seems to make a hash of some of our parser code.  It seems like it now has the wrong idea about the precedence of `<|>` vs `<$>`.  For example:
```
-        string "0b" *> L.binary
-          <|> string "0o" *> L.octal
-          <|> string "0x" *> L.hexadecimal
-          <|> L.decimal
+        string "0b"
+          *> L.binary
+            <|> string "0o"
+          *> L.octal
+            <|> string "0x"
+          *> L.hexadecimal
```

**Comment by byorgey** [comment]
@kostmo what do you think?  I think this is ready to go.  Once we rebase on top of #1367 it should build cleanly, I think.

**Comment by kostmo** [comment]
> @kostmo what do you think? I think this is ready to go. Once we rebase on top of #1367 it should build cleanly, I think.

Yes, just need your approval to merge.

**Comment by byorgey** [comment]
> Yes, just need your approval to merge.

Sure, since I made some changes too I wanted you to take a look.  I guess I could have just approved it and then let you merge it. :smile:

**Review by byorgey — APPROVED** [review]

**Pull request merged**